styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
```

Comment:
We shouldn't have nested containers.  You probably can just remove the `container-fluid` from the first div and be okay although you may need a row declaration in here somewhere as well.